bucket/goneovim.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "v0.4.12",
+    "description": "Neovim GUI written in Golang, using a Golang qt backend",
+    "url": "https://github.com/akiyosi/goneovim/releases/download/v0.4.12/Goneovim-v0.4.12-win64.zip",
```

Comment:
Could you add URL and Hash to architecture 64bit block?

This ensures the users using 32 bit are not inconvenienced

Reply:
Check https://github.com/ScoopInstaller/Extras/blob/master/bucket/staxrip.json for example